examples/resources/gitlab_runner/resource.tf

```diff
@@ -0,0 +1,21 @@
+# Basic GitLab Runner
```

Comment:
I know that we don't do this a lot (we gotta start somewhere 😛 )  ...

Thus, what do you think about providing a more extensive example which would someone incorporate the `authentication_token` into a `config.toml` template? (I assume that would be an example workflow with this resource, right?)

Reply:
Sure, I'll see what I can do.

Reply:
I did add an example that starts generating the config.toml, but with the way we have terraform-doc set up right now, the only thing we can "inject" is the example markdown. I could potentially create a new runner.md.tmpl file, and inject a new "Comprehensive Example" section, because it does feel like this is outside the scope of the basic example. I have to learn a bit more about the terraform-doc templating engine though to do that. I'd suggest that for the comprehensive example, I create a separate issue/PR for that 🙂